harness/determined/deploy/gke/constants.py

```diff
@@ -0,0 +1,9 @@
+class defaults:
+    GPU_TYPE = "nvidia-tesla-k80"
+    GPUS_PER_NODE = 4
+    ZONE = "us-west1-b"
+    MASTER_MACHINE_TYPE = "n1-standard-16"
+    AGENT_MACHINE_TYPE = "n1-standard-32"
+    MAX_GPU_NODES = 4
+    MAX_CPU_NODES = 4
+    K8S_NVIDIA_DAEMON = "https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml"
```

Comment:
Possibly the most useless code review comment of all time: there's a bunch of possibility for this floating around, it's badly documented which one you should use for which versions, etc., but this one is by far the best one to be using as far as I know. So... good job? :smile: